networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/topics/topichandlers/Eth2TopicHandler.java

```diff
@@ -130,7 +130,7 @@ private String getTopicName() {
   protected ValidationResult handleMessageProcessingError(Throwable err) {
     final ValidationResult response;
     if (ExceptionUtil.getCause(err, DecodingException.class).isPresent()) {
-      LOG.trace("Received malformed gossip message on {}", getTopic());
+      LOG.warn("Received malformed gossip message on {}", getTopic());
```

Comment:
Dropped this back to debug.  It should be rare but experience with besu is that these kinds of warnings can be a surprisingly effective way to fill a hard drive because they can happen quite frequently.  I've logged #4166 to make it easy to enable logging of these cases so that we're more aware of them and have more data always available.